virtualizarr/readers/hdf/hdf.py

```diff
@@ -58,6 +62,111 @@
 
 
 class HDFVirtualBackend(VirtualBackend):
+    @staticmethod
+    def _construct_manifest_array(
```

Comment:
Every `staticmethod` here aside from `open_virtual_dataset` may as well just be an actual function. You will need to do that anyway later to refactor to make the reader just one function.

Reply:
But you could leave that to do in #498.

Reply:
👍 That is the plan.  I think @maxrjones intent here was to make a very small change set to demonstrate the feasibility of using `ManifestStore` and that we would refactor the structure in a PR for https://github.com/zarr-developers/VirtualiZarr/issues/498.